src/workloads/execution/ExpressiveQueries.yml

```diff
@@ -0,0 +1,207 @@
+            state: {^Choose: {from: ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE", "DC", "FL", "GA", "HI", "ID", "IL", "IN", "IA", "KS", "KY", "LA", "ME", "MD", "MA", "MI", "MN", "MS", "MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY", "NC", "ND", "OH", "OK", "OR", "PA", "RI", "SC", "SD", "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI", "WY"]}},
+            zip: {^RandomString: {length: 6}},
+            location: {position: {type: "Point", coordinates: [
+              {^RandomDouble: {min: -118.668469, max: -82.062023}},
+              {^RandomDouble: {min: 32.924436, max: 41.679986}}
+            ]}}
+          }
+        },
+        {
+          policyType: "life",
+          policyNum: {^RandomString: {length: 30}},
+          nextRenewalDt: {^RandomDate: {min: "2017-01-01", max: "2017-12-31"}},
+          insured_person: {
+            firstname: {^RandomString: {length: {^RandomInt: {min: 3, max: 10}}}},
+            lastname: {^RandomString: {length: {^RandomInt: {min: 5, max: 10}}}},
+            yob: {^RandomDate: {min: "1930-01-01", max: "2016-12-31"}},
+            smoking: {^RandomInt: {min: 0, max: 1}}
+          }
+        }
+      ]}}}}
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: ManageDatabase
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        fsync: 1
+  - *Nop
+  - *Nop
+  - Repeat: 5
+    # Duration: 60 seconds
+    Database: *DB
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        createIndexes: Collection0
+        indexes:
+        - key:
+            address.state: 1
+            policies.policyType: 1
+            policies.insured_person.smoking: 1
+            gender: 1
+            dob: 1
+          name: usefulIndex
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        fsync: 1
+  - *Nop
+  - Repeat: 1
+    Database: *DB
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        drop: Collection0
+
+- Name: RunQueries
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: 10
+    Database: *DB
+    Operations:
+    - OperationMetricsName: QueryNoProjectionNoIndex
+      OperationName: RunCommand
+      OperationCommand:
+        find: Collection0
+        filter: {
+          gender: "Female",
+          dob: {
+            $gte: "1990-01-01",
+            $lte: "1990-12-31"
+          },
+          address.state: "UT",
+          policies: {
+            $elemMatch: {
+              policyType: "life",
+              insured_person.smoking: 1
+            }
+          }
+        }
+  - Repeat: 10
+    Database: *DB
+    Operations:
+    - OperationMetricsName: QueryWithProjectionNoIndex
+      OperationName: RunCommand
+      OperationCommand:
+        find: Collection0
+        filter: {
```

Comment:
If you define &filter anchor above, here you can replace lines 168-181 with
filter: *filter

It also means you can guarantee your queries use exactly the same filter.